lib/matplotlib/artist.py

```diff
@@ -181,6 +181,7 @@ def __init__(self):
         self._path_effects = mpl.rcParams['path.effects']
         self._sticky_edges = _XYPair([], [])
         self._in_layout = True
+        super().__init__(**kwargs)
```

Comment:
Prob needs a comment here, because its a bit strange to super().__init__ in the init of a base class.